src/screens/ExchangeModal.js

```diff
@@ -66,17 +64,12 @@ const FloatingPanels = ios
   ? AnimatedExchangeFloatingPanels
   : ExchangeFloatingPanels;
 
-const Wrapper = ios ? KeyboardFixedOpenLayout : Fragment;
+const Wrapper = KeyboardFixedOpenLayout;
```

Comment:
could remove this but want to keep as im going to refactor `keyboardFIxedOpenLayout` soon: see RNBW-2275